File: P5U4.cs
using System;

class Program
{
    static void Main()
    {
        
        string[] frutas = { "Manzana", "Fresa", "Platano", "Melon", "Uvas" };

        
        string[] frutas_AlReves = new string[frutas.Length];

       
        for (int i = 0; i < frutas.Length; i++)
        {
            frutas_AlReves[i] = frutas[frutas.Length - 1 - i];
        }

       
        Console.WriteLine("Arreglo Original:");
        ImprimirArreglo(frutas);

        
        Console.WriteLine("\nArreglo al Revés:");
        ImprimirArreglo(frutas_AlReves);
    }

    static void ImprimirArreglo(string[] array)
    {
        foreach (string elemento in array)
        {
            Console.Write("{elemento} ");
        }
        Console.WriteLine();
    }
}

File: P5U4.java
package com.mycompany.p5u4;

public class P5u4 {

    public static void main(String[] args) {
        String[] frutas = {"fres", "Mango", "uvas", "manzana", "melon","sandia"};


        String[] frutasAlReves = new String[frutas.length];


        for (int i = 0; i < frutas.length; i++) {
            frutasAlReves[i] = frutas[frutas.length - 1 - i];
        }


        System.out.println("Arreglo Original:");
        imprimirArreglo(frutas);


        System.out.println("\nArreglo al Revés:");
        imprimirArreglo(frutasAlReves);
    }


    static void imprimirArreglo(String[] arreglo) {
        for (String elemento : arreglo) {
            System.out.print(elemento + " ");
        }

    }
}    